src/elements/content-sidebar/activity-feed/activity-feed/EmptyState.scss

```diff
@@ -0,0 +1,26 @@
+.bcs-EmptyState {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    .bcs-EmptyState-cta {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        aside {
```

Comment:
Ideally, this should use an non-nested class name.

Reply:
Ah, good catch, let me fix that